### PR TITLE
Modify Wildcard#match(String, String) into #match(CharSequence, CharSequence)

### PR DESCRIPTION
**Pull request by flowbehappy**
For wider applicability, like CharBuffer etc.

### CODE REVIEW

**Comment by igorspasic** [comment]
Awesome, just in time! We might change other string-related utils as well!

**Comment by igorspasic** [comment]
Just started reviewing :)<br>:rocket:

**Comment by slandelle** [comment]
@igorspasic Preparing for JDK9 String? ;)

**Comment by igorspasic** [comment]
will see @slandelle  :)))

**Comment by slandelle** [comment]
I don't have the cycles to help atm, but I could be able to help in January.

**Comment by igorspasic** [comment]
No worries, its far from being urgent...

**Comment by slandelle** [comment]
Balance between fun and far-from-urgent tasks and most pressing headache giving ones is very delicate...

**Comment by igorspasic** [comment]
Can I quote this sentence!? :)

**Comment by slandelle** [comment]
In your performance evaluation? Sure!

File: jodd-core/src/main/java/jodd/util/Wildcard.java [hunk 1]
```diff
@@ -138,7 +138,7 @@ private static boolean match(String string, String pattern, int sNdx, int pNdx)
 	/**
 	 * Matches string to at least one pattern.
 	 * Returns index of matched pattern, or <code>-1</code> otherwise.
-	 * @see #match(String, String)
+	 * @see #match(CharSequence, CharSequence)
 	 */
```
Comment (septs) [inline jodd-core/src/main/java/jodd/util/Wildcard.java]: be ignored?

Comment (igorspasic) [inline jodd-core/src/main/java/jodd/util/Wildcard.java]: Sorry, what do you think that should be ignored? I am not sure if I got it ;)

**Comment by flowbehappy** [comment]
@igorspasic Great!

**Pull request merged**